start-from-zero/Javascript/src/views/evaluation/evaluation.js

```diff
@@ -1,145 +1,144 @@
-windowObjectWrap.EvaluationsPage = function(options){
-    const headings = ['Name','Language','Level','Detalii'];
-
-    const localSt = localStorage.length;
-    var rows;
-    if(localSt == 1){
-        rows = [];
-    }else{
-        rows = JSON.parse(localStorage.getItem("localArr"));
+(function(){
+    windowObjectWrap.EvaluationsPage = function(options){
+        const headings = ['Name','Date','Level','Detalii'];
+    
+        const localSt = localStorage.length;
+        var rows;
+        if(localSt == 1){
```

Comment:
Are you absolutely sure that your app will only ever store a single piece of information in local storage, and that said piece of information will always be `localArr`?